### PR TITLE
Added download function to download models

### PR DESCRIPTION
**Pull request by KapilBansal320**

### CODE REVIEW

File: freshlybuiltimagebol/downloader.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			if ch==1:
+			response = requests.get(url)
+			if response.status_code==200:
+				with open(output_model, 'wb') as file:
+					file.write(response.content)
+
+	def models(self):
+		self.model={
+				'frozen_east_text_detection':"https://github.com/ZER-0-NE/EAST-Detector-for-text-detection-using-OpenCV/blob/master/frozen_east_text_detection.pb?raw=true"
+				}
```
Comment (vishal2612200) [inline freshlybuiltimagebol/downloader.py:23]: Use string operation for the addition of model name in URL rather than a dictionary data structure.

Comment (KapilBansal320) [reply freshlybuiltimagebol/downloader.py:23]: Done

File: freshlybuiltimagebol/downloader.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import requests
+from os.path import isfile
+
+class Model_File():
+	#Download the model
+	def __init__(self,model_name):
```
Comment (vishal2612200) [inline freshlybuiltimagebol/downloader.py:6]: Add multiline docstring to the function.

File: freshlybuiltimagebol/downloader.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import requests
```
Comment (vishal2612200) [inline freshlybuiltimagebol/downloader.py:1]: Use from statement instead of import

**Pull request merged**